erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -617,10 +617,12 @@ def get_party_details(company, party_type, party, date):
 
 	account_currency = get_account_currency(party_account)
 	account_balance = get_balance_on(party_account, date)
+	party_name = frappe.db.get_value(party_type, party, party_name)
```

Comment:
Is this working? It should give error `party_name is not defined`

Reply:
My bad! Fixed it.